packages/css-library/dist/tokens/scss/variables.scss

```diff
@@ -190,6 +190,64 @@ $units-neg-8: -4rem;
 $units-neg-9: -4.5rem;
 $units-neg-10: -5rem;
 $units-neg-15: -7.5rem;
+$vads-spacing-0: 0;
+$vads-spacing-1: 8px;
+$vads-spacing-2: 16px;
+$vads-spacing-3: 24px;
+$vads-spacing-4: 32px;
+$vads-spacing-5: 40px;
+$vads-spacing-6: 48px;
+$vads-spacing-7: 56px;
+$vads-spacing-8: 64px;
+$vads-spacing-9: 72px;
+$vads-spacing-10: 80px;
+$vads-spacing-15: 120px;
+$vads-spacing-1px: 1px;
+$vads-spacing-2px: 2px;
+$vads-spacing-0p5: 4px;
+$vads-spacing-1p5: 12px;
+$vads-spacing-2p5: 20px;
+$vads-spacing-neg-1px: -1px;
+$vads-spacing-neg-2px: -2px;
+$vads-spacing-neg-0p5: -4px;
+$vads-spacing-neg-1: -8px;
+$vads-spacing-neg-1p5: -12px;
+$vads-spacing-neg-2: -16px;
+$vads-spacing-neg-2p5: -20px;
+$vads-spacing-neg-3: -24px;
+$vads-spacing-neg-4: -32px;
+$vads-spacing-neg-5: -40px;
+$vads-spacing-neg-6: -48px;
+$vads-spacing-neg-7: -56px;
+$vads-spacing-neg-8: -64px;
+$vads-spacing-neg-9: -72px;
+$vads-spacing-neg-10: -80px;
+$vads-spacing-neg-15: -120px;
+$vads-spacing-inset-l: 32px;
+$vads-spacing-inset-m: 24px;
+$vads-spacing-inset-s: 16px;
+$vads-spacing-vertical-stack-l: 32px;
+$vads-spacing-vertical-stack-m: 24px;
+$vads-spacing-vertical-stack-s: 16px;
+$vads-spacing-vertical-stack-default: 16px;
+$vads-forms-spacing-vertical-stack-default: 32px;
+$uswds-system-spacing-special-0: 0;
```

Comment:
Did it seems like a simple thing to omit these `uswds-system` prefixed tokens? Or is that going to be a little heavier lift than we think?

Reply:
I was thinking about including it in this ticket, we'll just have to remember to update the uswds- tokens that are being used in vets-website.